<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>powder_calibration_dialog</class>
 <widget class="QDialog" name="powder_calibration_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>163</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="eta_tolerance">
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>2.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="peak_fit_type">
       <item>
        <property name="text">
         <string>PVoigt</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gaussian</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="tth_tolerance">
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.200000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Peak fitting type:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="tth_label">
       <property name="text">
        <string>2θ tolerance:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="eta_label">
       <property name="text">
        <string>η tolerance:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QCheckBox" name="robust">
       <property name="text">
        <string>Use robust optimization?</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tth_tolerance</tabstop>
  <tabstop>eta_tolerance</tabstop>
  <tabstop>peak_fit_type</tabstop>
  <tabstop>robust</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>powder_calibration_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>powder_calibration_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
